client/example-docker.yaml

```diff
@@ -2,7 +2,7 @@ build:
   base_image: python:3.11-slim
   dependencies:
     apt: [wget]
-    pip: [-r requirements.txt, .]
+    pip: [-r example-requirements.txt, .]
```

Comment:
Maybe this is a good opportunity to have two distinct requirement files (and thus image build specs) for the MNIST and Hello World jobs, so that we don't need to include Tensorflow for HW.